SingularityUI/app/templates/vex/requestRun.hbs

```diff
@@ -12,3 +12,34 @@
 <div class="vex-dialog-input">
 	<input id='filename' name="filename" type="text" class="vex-dialog-prompt-input" placeholder="e.g. service.log" value="">
 </div>
+
+<script>
```

Comment:
give me a shout if you get stuck - there's a few different ways to use vex, some more shorthand than others (you can see different examples throughout singularity) and the vex docs obviously as well could be useful to read through http://github.hubspot.com/vex/